lib/rspec-console.rb

```diff
@@ -1,7 +1,10 @@
 module RSpecConsole
   autoload :ConfigCache, 'rspec-console/config_cache'
+  autoload :Environment, 'rspec-console/environment'
+  autoload :Proxy,       'rspec-console/proxy'
   autoload :Runner,      'rspec-console/runner'
   autoload :Pry,         'rspec-console/pry'
+  autoload :VersionError,'rspec-console/exceptions'
```

Comment:
`VersionError` should be `Exception` (and the exceptions.rb file should be renamed to exception.rb)